data/json/recipes/armor/torso.json

```diff
@@ -197,11 +220,13 @@
     "skill_used": "tailor",
     "skills_required": [ "survival", 5 ],
     "difficulty": 5,
-    "time": "3 h 20 m",
+    "time": "10 h",
     "autolearn": true,
     "qualities": [ { "id": "SEW", "level": 1 }, { "id": "CUT", "level": 1 } ],
     "book_learn": [ [ "trappers_companion", 6 ], [ "survival_book", 4 ] ],
-    "components": [ [ [ "stomach_sealed", 16 ], [ "large_stomach_sealed", 10 ] ], [ [ "filament", 60, "LIST" ] ] ]
+    "using": [ [ "fastener_large", 1 ] ],
```

Comment:
In this case it is probably held together by the filament, this is a very pre-industrial kind of coat and I doubt it even has buttons.